plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/config/Config.java

```diff
@@ -61,6 +65,30 @@ public void setPlugins(List<Plugin> plugins) {
         return plugins;
     }
 
+    public void setJenkinsUc(String jenkinsUc) {
```

Comment:
It would be good to just pass a config object to the constructor here then there would be no need to copy the config over in setters,  I think it would be simpler. not required in this PR though

Reply:
I think that's actually what I had originally and Kristin suggested having the setup method return a fully configured Config class.